openage/convert/changelog.py

```diff
@@ -35,7 +35,7 @@
     {"graphics"},
     {"interface"},
     {"interface"},
-    {"gamedata"},
+    {"gamedata", "metadata"},
```

Comment:
Yes I saw that, I now am not sure to use the `targetdir` in deleting the folder:
```python
shutil.rmtree(args.targetdir ???? 'gamedata')
```